UI/Web/src/app/app.module.ts

```diff
@@ -14,6 +14,8 @@ import { SidenavModule } from './sidenav/sidenav.module';
 import { NavModule } from './nav/nav.module';
 
 
+// Disable Web Animations if the user's browser (such as iOS 12.5.5) does not support this.
+const disableAnimations = !('animate' in document.documentElement);
```

Comment:
I think we should do a console.error('Animations have been disabled as your browser doesn't support them'); or something to inform the user, even if most wont see this.